cmd/jiralert/content.go

```diff
@@ -99,6 +99,12 @@ func pageTemplate(name string) *template.Template {
 // HomeHandlerFunc is the HTTP handler for the home page (`/`).
 func HomeHandlerFunc() func(http.ResponseWriter, *http.Request) {
 	return func(w http.ResponseWriter, r *http.Request) {
+		if r.Method != "GET" {
```

Comment:
This makes sense, but I wonder why you had no response or misleading error for this?

Reply:
What I basically wanted here is: send an error-statuscode when someone tries to POST on /

When i configured `http://jiralert:9097/` (without `/alert`) Alertmanager happily POSTed it's webhook notifications to / and did not error because jiralert responded with a `200` statuscode.

Reply:
Makes sense!